src/game.cpp

```diff
@@ -7204,8 +7204,8 @@ void game::open()
                 // curtains as well.
                 int outside_openable = veh->next_part_to_open(vpart, true);
                 if (outside_openable == -1) {
-                    const char *name = veh->part_info( openable ).name().c_str();
-                    add_msg(m_info, _("That %s can only opened from the inside."), name);
+                    const auto name = veh->part_info( openable ).name();
```

Comment:
Why ```auto``` and not ```std::string```? I remember it was discussed before that _excessive_ using of ```auto``` makes code harder to understand for newbies.